lexicons/com/atproto/admin/defs.json

```diff
@@ -167,9 +168,18 @@
           "type": "string",
           "format": "datetime"
         },
+        "lastAppealedAt": {
+          "type": "string",
+          "format": "datetime",
+          "description": "Timestamp referencing when the owner of the subject appealed a moderation action"
```

Comment:
It is a small nit (could ignore this comment), but "owner" feels subtly weird to me here. We use "author" just below, that feels better. Or "creator".